src/rgw/rgw_data_sync.cc

```diff
@@ -4401,6 +4471,17 @@ int RGWBucketShardIncrementalSyncCR::operate()
                 return 0;
               });
       }
+
+      if (!truncated && extended_result.next_log) {
+        yield call(new RGWBucketShardIsDoneCR(sc, status_oid, num_shards, objv_tracker));
```

Comment:
this instance of `objv_tracker` is the one we're using for the _shard's_ sync status object, so we shouldn't reuse it for the bucket-wide object. since `RGWBucketShardIsDoneCR` is doing the read-modify-write internally, it should have its own instance of `objv_tracker` as a member variable